forge-gui/res/tokenscripts/w_2_2_knight_protectionred.txt

```diff
@@ -0,0 +1,7 @@
+Name:Knight Token
```

Comment:
please name the file w_2_2_knight_pro_red

Reply:
Done !